go/src/koding/klientctl/install-kd.sh

```diff
@@ -102,6 +106,40 @@ install_vagrant_deps() {
   fi
 }
 
+test_vagrant() {
+  echo "VirtualBox and Vagrant versions must match in order" 1>&2
+  echo "to be able to build a Vagrant box." 1>&2
+  echo 1>&2
+  echo "Vagrant 1.7.x expects VirtualBox 4.x versions and" 1>&2
+  echo "Vagrant 1.8.x expects VirtualBox 5.x versions." 1>&2
+  echo 1>&2
+
+  if ! prompt_install "Do you want to test building Vagrant box with VirtualBox provider?"; then
```

Comment:
Ran into an issue here i think, my script didn't continue installing vagrant - should it?

_(I'm talking with Rafal now about it, just didn't want an accidental merge if there's an issue, so documenting it here)_